test/fixture/name-of-file-to-load.egi

```diff
@@ -0,0 +1 @@
+def x := 1
```

Comment:
https://github.com/egison/egison/blob/8ebeacaa37835f4ff5703f4277f25c9165001021/docs/reference/command-line-options.rst#L11-L12

write `x := 1` in document.
but, occur error

```
$ stack exec -- egison -l test/fixture/name-of-file-to-load.egi 
Parse error at: egison:1:3:
  |
1 | x := 1
  |   ^
unexpected ':'
expecting "...", "where", atomic expression, end of input, index, or infix
```

should I fix documents in this PR? 
or, different PR?

Reply:
Right, we changed the syntax from v4.1.0 to require `def` keyword before global definitions, but we haven't updated the docs yet.
Feel free to open another PR to fix this (if you have time, of course)